acceptance/acceptance_test.go

```diff
@@ -313,9 +314,17 @@ func getEnvFilters(t *testing.T) []string {
 		key := items[0]
 		// Clear it just to be sure, since it's going to be part of os.Environ() and we're going to add different value based on settings.
 		os.Unsetenv(key)
+
+		if key == "DATABRICKS_CLI_DEPLOYMENT" && items[1] == "direct" {
+			// CLI only recognizes "direct-exp" at the moment, but in the future will recognize "direct" as well.
+			// On CI we set "direct". To avoid renaming jobs in CI on the future, we correct direct -> direct-exp here
```

Comment:
```suggestion
			// On CI we set "direct". To avoid renaming jobs on CI in the future, we correct direct -> direct-exp here
```

Reply:
good correction, missed this before merging.